app/assets/javascripts/discourse/templates/review-index.hbs

```diff
@@ -17,17 +17,37 @@
   </div>
 
   <div class='reviewable-filters'>
-    <div class='reviewable-filter'>
-      <label class='filter-label'>{{i18n "review.filters.status"}}</label>
-      {{combo-box
-        value=filterStatus
-        content=statuses
-        onChange=(action (mut filterStatus))
-      }}
+    <div class='reviewable-filter quick-filters'>
+      {{i18n "review.filters.quick"}}
+      <div>
+        {{d-button
+          label="review.filters.reset"
+          icon="times"
+          class="btn-default"
+          disabled=disableFilterReset
+          action=(action "resetFilters")}}
+
+        {{plugin-outlet name="review-quick-filters" args=(hash model=model)}}
+
+        {{d-button
+          label="review.filters.resolved"
+          class="btn-default"
+          action=(action "setFilters")
+          actionParam='{"sort_order": "created_at", "status": "reviewed"}'}}
```

Comment:
How do I pass an object in from a Handlebars template?

Reply:
```
actionParam=(hash foo=1 bar=2)
```